app/assets/javascripts/discourse/app/controllers/invites-show.js

```diff
@@ -26,11 +29,13 @@ export default Controller.extend(
     errorMessage: null,
     userFields: null,
     inviteImageUrl: getUrl("/images/envelope.svg"),
+    isInviteLink: alias("model.is_invite_link"),
+    rejectedEmails: A(),
```

Comment:
Why do you need a native array here ?

Also I would prefer this to be null, and the array initiated in init()

Reply:
I basically copied the same implementation we have in [`create-account.js`](https://github.com/discourse/discourse/blob/c014b9385448a67143ef401d5461681caa49633c/app/assets/javascripts/discourse/app/controllers/create-account.js#L36).

I made the changes you suggested here.